packages/material-ui/src/useMediaQuery/useMediaQuery.d.ts

```diff
@@ -16,4 +16,7 @@ export interface Options {
   ssrMatchMedia?: (query: string) => MuiMediaQueryList;
 }
 
-export default function useMediaQuery(query: string, options?: Options): boolean;
+export default function useMediaQuery(
+  query: string | ((theme: any) => string),
```

Comment:
```suggestion
  query: string | ((theme: Theme) => string),
```

Reply:
Multi-line suggestions would be nice

Reply:
What's multi-line suggestions?

Reply:
Oh, I think that I get it.

Reply:
I had to make two comments for this suggestion, because a suggestion can't span multiple lines:

```diff
-export default function useMediaQuery(
+export default function useMediaQuery<Theme = unknown>(
-  query: string | ((theme: any) => string),
+  query: string | ((theme: Theme) => string),
  options?: Options,
): boolean;
```